HaskellDates.hsproj/_HaskellDates.hsplay

```diff
@@ -0,0 +1,41 @@
+-- Haskell Playground 1.0
+now <- utctDay <$> getCurrentTime
+weekDay now
+
+let eventDay = addDays (-27) now
+weekDay eventDay
+
+let event = Event { date = eventDay, daysOfWeek = [3,5], week = -1 }
+
+let startDay = addDays (-15) now
+let endDay = addDays 15 now
+
+
+let allDays = takeWhile (<=endDay) $ (`addDays` startDay) <$> [0..]
+let validDays = filter (((flip elem) $ daysOfWeek event) . dayOfWeek) allDays
```

Comment:
maybe? It looks like `intersect` is `[a] -> [a] -> [a]` but I have 2 different types. I need `[Day] -> [Int] -> [Day]`. I could map the `allDays` array to get the corresponding `Int` but then I would have to get the day back after the intersect.